providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -524,10 +524,10 @@ def consume_logs(*, since_time: DateTime | None = None) -> tuple[DateTime | None
                                 message_timestamp = line_timestamp
                                 progress_callback_lines.append(line)
                             else:  # previous log line is complete
-                                for line in progress_callback_lines:
+                                for progress_line in progress_callback_lines:
```

Comment:
That does not seem like an eror. It's just a variable name. Are you sure that if you come back to the original code your test is failing ? It should not.